NetCDFIO.pyx

```diff
@@ -59,9 +60,11 @@ cdef class NetCDFIO_Stats:
 
         Pa.barrier()
 
+
+
         if Pa.rank == 0:
             shutil.copyfile(
-                os.path.join( './', namelist['meta']['simname'] + '.in'),
+                os.path.join( './', namelist['meta']['simname'] + '.in'), # 111018[ZS]
```

Comment:
```suggestion
                os.path.join( './input/', namelist['meta']['simname'] + '.in'),
```
(I added an input folder as there are too many input files for the GCM driven cases.)